src/components/seo.js

```diff
@@ -72,7 +77,14 @@ const SEO = ({
           name: 'twitter:description',
           content: metaDescription,
         },
-      ].concat(meta)}
+      ].concat(
```

Comment:
instead of concatenating, it can be moved to the above array since it's a single element

Reply:
Yeah👍🏽